### PR TITLE
added teradata.mdx file

### PR DESCRIPTION
**Pull request by happysingh222**
## Description

Create a doc page for the Teradata data integration

**Fixes** #5148 

## Type of change

(Please delete options that are not relevant)

- [x] 📄 This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have updated the documentation, or created issues to update them.

### CODE REVIEW

**Comment by happysingh222** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by happysingh222** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/app-integrations/teradata.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+* `user`: specifies the user name
+* `password`: specifies the password for the user
+* `database`: sets the database for the connection
+
+## Usage
+
+Assuming you created a database in Teradata called `HR` and you have a table called `Employees` that was created using
+the following SQL statements:
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/app-integrations/teradata.mdx:30]: Please use the below code block:

> \```sql
> SQL GOES HERE
> \```

File: docs/app-integrations/teradata.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        'Adam',
+        'Tworkowski',
+        '1980-01-05',
+        '2004-08-01',
+        01);
+~~~~
+
+In order to make use of this handler and connect to the Teradata database in MindsDB, the following syntax can be used:
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/app-integrations/teradata.mdx:63]: Please use the below code block:

> \```sql
> SQL GOES HERE
> \```

File: docs/app-integrations/teradata.mdx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    "database": "HR"
+};
+
+~~~~
+
+**Note**: The above example assumes usage of Teradata Vantage running on Oracle VirtualBox.
+
+Now, you can use this established connection to query your database as follows:
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/app-integrations/teradata.mdx:79]: Please use the below code block:

> \```sql
> SQL GOES HERE
> \```

**Comment by martyna-mindsdb** [comment]
@happysingh222 
Please implement the above review comments in this PR.

**Comment by happysingh222** [comment]
@martyna-mindsdb Please review my pr. Thankyou

**Pull request merged**